Makefile

```diff
@@ -56,9 +56,11 @@ NAMESPACE ?= istio-system
 GOPATH ?= ${HOME}/go
 
 # Environment variables set when running the Go compiler.
+GOOS ?= $(shell go env GOOS)
+GOARCH ?= $(shell go env GOARCH)
```

Comment:
I just want to make sure this is going to work when Travis and Jenkins performs our builds.
What if the build machine reports different GOOS and GOARCH? (i.e. are Travis/Jenkins build machines linux/arch64?)

Reply:
Same as below, we still need to discuss how we build the multiarch image (by either cross building or building on arm64 natively). But the idea of this change is: it allows the buiding can be carried out on aarch64, without affecting the current building on x86_64. I think the CI machine is x86_64 only, but by trying configuation in https://blog.travis-ci.com/2019-10-07-multi-cpu-architecture-support, we can build on both x86_64 and aarch64 machines by applying this change here.

Reply:
Travis reports GOOS=linux and GOARCH=amd64. (See first and last lines in highlighted range https://travis-ci.org/github/kiali/kiali/builds/664245023#L182)

Jenkins also reports the same. This looks safe.